bindings/python/iota_sdk/wallet/sync_options.py

```diff
@@ -7,40 +7,40 @@ class AccountSyncOptions():
     Attributes:
         basic_outputs: Whether to sync basic outputs.
         nft_outputs: Whether to sync NFT outputs.
-        alias_outputs: whether to sync alias outputs.
+        account_outputs: whether to sync account outputs.
     """
 
     def __init__(self,
                  basic_outputs: Optional[bool] = None,
                  nft_outputs: Optional[bool] = None,
-                 alias_outputs: Optional[bool] = None):
+                 account_outputs: Optional[bool] = None):
         """Initialize `Self`.
         """
         self.basicOutputs = basic_outputs
         self.nftOutputs = nft_outputs
-        self.aliasOutputs = alias_outputs
+        self.accountOutputs = account_outputs
```

Comment:
Unrelated but didn't we want to have everything snake_case?

Reply:
Yes looks like this was forgotten

Reply:
Another PR of you want to address it here?

Reply:
Would rather do it in a new one, created an issue https://github.com/iotaledger/iota-sdk/issues/1153

Reply:
Sounds good